src/app/shared/components/delegated-acc-message/delegated-acc-message.component.ts

```
@@ -1,20 +1,39 @@
+import { Router } from '@angular/router';
+import { NetworkService } from '../../../core/services/network.service';
 
 @Component({
   selector: 'app-delegated-acc-message',
   templateUrl: './delegated-acc-message.component.html',
   styleUrls: ['./delegated-acc-message.component.scss'],
 })
 export class DelegatedAccMessageComponent implements OnInit {
-  delegateeName;
+  delegateeName: string;
+  isConnected$: any;
```

ESLint

[error] Expected blank line between class members.

(lines-between-class-members)